.ci_support/linux_64_numpy1.21python3.8.____cpython.yaml

```diff
@@ -14,6 +14,8 @@ cxx_compiler_version:
 - '12'
 docker_image:
 - quay.io/condaforge/linux-anvil-cos7-x86_64
+numpy:
+- '1.21'
```

Comment:
do notice that numpy is pinned at 1.21 at build time. At run time, numpy becomes pinned to 1.21,<2